docs/conf.py

```diff
@@ -128,12 +128,12 @@ def setup(app):
 
 redirects = {
     "api-pytorch": "training/apis-howto/api-pytorch-ug.html",
```

Comment:
you need to be adding a redirect from the deleted `api-core-ug.html` to the new `api-core-mnist-ug.html`.

Or alternatively, if the document is literally replaing the old one... why bother with renames or redirects?  Why not just replace the content in `api-core-ug.html`?

Reply:
renamed api-core-mnist-ug to api-core-ug